SW_Flow_lib.c

```diff
@@ -1056,9 +1056,8 @@ void hydraulic_redistribution(double swc[], double swcwp[], double lyrRootCo[],
 	 **********************************************************************/
 
 	unsigned int i, j;
-	double swp[nlyrs], swpwp[nlyrs], relCondroot[nlyrs], hydredmat[nlyrs][nlyrs], Rx, swa,
-		hydred_sum, x;
-
+	double swp[OVER_SIZE] = {0}, swpwp[OVER_SIZE] = {0}, relCondroot[OVER_SIZE] = {0}, hydredmat[OVER_SIZE][OVER_SIZE] = {{0}};

```

Comment:
Why not use existing constant for this specific purpose of soil layer number, i.e., `MAX_LAYERS`, if we want here a constant array size?